src/ripple/protocol/impl/STObject.cpp

```diff
@@ -35,7 +35,7 @@ STObject::~STObject()
 #endif
 }
 
-STObject::STObject(STObject&& other)
+STObject::STObject(STObject&& other) noexcept
```

Comment:
`CountedObject<STObject>` is not noexcept default constructible.

Reply:
Fixed in 209c9fd7ad71ce26b868c1df276ff184f89f6660